workflow/util/util.go

```diff
@@ -764,8 +764,8 @@ func getDescendantNodeIDs(wf *wfv1.Workflow, node wfv1.NodeStatus) []string {
 	for _, child := range node.Children {
 		childStatus, err := wf.Status.Nodes.Get(child)
 		if err != nil {
-			log.Fatalf("Couldn't get child, panicking")
-			panic("Was not able to obtain child")
+			log.Warnf("Coudn't obtain child for %s, panicking", child)
+			panic("was not able to obtain child")
```

Comment:
I was gonna recommend outputting more details here too 👍 

also apparently there is a shorthand for this as well: [`log.Panic`](https://pkg.go.dev/log#Panic)

```suggestion
			log.Panicf("Coudn't obtain child for %s, panicking", child)
```

also `Error` is the appropriate log level here if we're panicking, vs. `Warn`